ocm/ocm-overview.adoc

```diff
@@ -0,0 +1,44 @@
+[id="ocm-overview"]
+= Red Hat OpenShift Cluster Manager (OCM)
+include::modules/common-attributes.adoc[]
+:context: ocm-overview
+toc::[]
+
+The Red Hat OpenShift Cluster Manager (OCM) allows users to create, manage, and delete OpenShift clusters using a web UI.
+
+include::modules/ocm-accessing.adoc[leveloffset=+1]
+
+[id="ocm-general-actions"]
+== General actions
+
+On the top right of the cluster page, there are some actions that a user can perform on the entire cluster.
+
+. **Open console** launches a web console so that the cluster owner can issue commands to the cluster.
+. **Actions** dropdown menu allows the cluster owner to rename the display name of the cluster, change the amount of load balancers and persistent storage on the cluster, if applicable, manually set the node count, and delete the cluster.
```

Comment:
There is another action in the dropdown with capability set, like "Hibernate cluster". Should it be mentioned here?

Reply:
@xueli181114, sure thing! What does the hibernate cluster action do specifically? Does it just pause the cluster until a user needs it? Any other considerations to add?

Reply:
@EricPonvelle  I agree with @wgordon17 's comments. Let's skip it.